tests/resources/functions/go-1.22/tests.go

```diff
@@ -0,0 +1,156 @@
+package handler
+
+import (
+	"openruntimes/types"
+	"os"
+)
+
+func Main(Context *types.Context) types.ResponseOutput {
+	action := Context.Req.Headers["x-action"]
+
+	switch a := action; a {
+	case "plaintextResponse":
+		return Context.Res.Send("Hello World 👋", 200, nil)
```

Comment:
Looking for review, is this typical in Golang?

I was considering `context.res.Send()` instead, but then I saw Go code example `os.Stdout = writer` which made me think variables are too first-uppercase. Not 100% confident.